recipes/server_check.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative './server_check_helper'
+
+consul_service_folder = '/etc/systemd/system/consul.service.d'
+drop_in_file = "#{consul_service_folder}/server_start_prevention.conf" # prevent systemd from starting
+
+directory consul_service_folder do
+  owner 'root'
+  mode '0644'
```

Comment:
Directories should have the execution bit

```suggestion
  mode '0755'
```